.gitignore

```diff
@@ -0,0 +1 @@
+/vendor/
```

Comment:
I think we've swapped sides on this issue.  Last time, you convinced me (rightly so, I feel) we should keep the composer.lock file: https://getcomposer.org/doc/01-basic-usage.md#composer-lock-the-lock-file